config/routes.rb

```diff
@@ -8,7 +8,7 @@
 
   ruby_versions = RubyConfig.ruby_versions.collect { |v| Regexp.escape(v.version) }
 
-  scope "(:version)", constraints: { version: /#{ruby_versions.join("|")}/ } do
+  scope "(:version)", constraints: { version: /#{ruby_versions.join("|")}|latest/ } do
```

Comment:
Since `RubyConfig.version_for("latest")` would return nil, no other change is needed.

https://github.com/rubyapi/rubyapi/blob/6fea025bf7626fedbf56d72489cb70d9154c4237/app/controllers/application_controller.rb#L8

Reply:
I would prefer to not rely on an unspecified rule as the way to support this feature. It's hard for anyone reading the code for the first time to understand this and it's very easy to break unintentionally.